Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/features/themeSlice.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+type THEME_VALUES = 'light' | 'dark';
+
+interface IThemeState {
+    mode: THEME_VALUES;
+}
+
+const initialState: IThemeState = { mode: 'dark' };
+
+const themeSlice = createSlice({
+    name: 'theme',
+    initialState,
+    reducers: {
+        toggleTheme: (state) => {
+            // eslint-disable-next-line no-unused-expressions
+            state.mode === 'light'
+                // eslint-disable-next-line no-param-reassign,prefer-destructuring
+                ? state.mode = 'dark'
+                // eslint-disable-next-line no-param-reassign,prefer-destructuring
+                : state.mode = 'light';
```

Comment:
Is it necessary to disable the rules? Isn't conforming possible?

Reply:
in rtk, one of the features is that it works with reassgning params as immutable, it does that for us and it was coded to work that way. There could be workaround, but it doesnt sound right to look for workaround for something that works right. A solution that comes to my mind is to have a naming convention for slice files and disable the rule for them: { "files": ["src/**/*.slice.ts"], "rules": { "no-param-reassign": "off" } }

Reply:
The proposed solution seems reasonable. Can you implement it and remove the comments?